### PR TITLE
Added unit test for HttpPut request in jsonrequesttest.java and added tests for textrequest.java

### PR DESCRIPTION
**Pull request by Nike682631**
See title

### CODE REVIEW

File: src/test/java/rest/TextRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        request = new JsonRequest(connection, httpGet);
+        Whitebox.setInternalState(request, "client", httpClient);
+
+        HttpPut httpPut = Mockito.mock(HttpPut.class);
+        request = new JsonRequest(connection, httpPut, "");
+        Whitebox.setInternalState(request, "client", httpClient);
+    }
+
+    @Test
+    public void testHttpGet_throws_exception() throws IOException {
```
Comment (frankgiordano) [inline src/test/java/rest/TextRequestTest.java:39]: I would like to follow camel style for methods names, how about:
testHttpGetThrowsExecption
or another short handle which I have seen in many production projects:
tstHttpGetThrowsExecption

**Comment by frankgiordano** [comment]
can you see my example about and change all the methods.. what do you think? thanks for the PR request looks good!

File: src/test/java/rest/JsonRequestTest.java [hunk 2]
```diff
@@ -31,18 +32,22 @@ public void init() {
 
         request = new JsonRequest(connection, httpGet);
         Whitebox.setInternalState(request, "client", httpClient);
+
+        HttpPut httpPut = Mockito.mock(HttpPut.class);
```
Comment (Nurkambay) [inline src/test/java/rest/JsonRequestTest.java:36]: This is not used. Please add tests got httpPut or remove this lines

**Review by Nurkambay — CHANGES_REQUESTED** [review]

File: src/test/java/rest/JsonRequestTest.java [hunk 3]
```diff
@@ -21,38 +19,39 @@
...
     }
 
     @Test
     public void tstHttpGetThrowsException() throws IOException {
         Mockito.when(httpClient.execute(any(HttpUriRequest.class), any(ResponseHandler.class)))
-               .thenThrow(new IOException());
+            .thenThrow(new IOException());
 
         assertThrows(IOException.class, request::httpGet);
-        Mockito.verify(httpClient).execute(any(HttpGet.class), any(BasicResponseHandler.class));
```
Comment (Nurkambay) [inline src/test/java/rest/JsonRequestTest.java:46]: Please revert these changes.

File: src/test/java/rest/JsonRequestTest.java [hunk 4]
```diff
@@ -21,38 +19,39 @@
...
     }
 
     @Test
     public void tstHttpGetReturnsNullForInvalidJson() throws IOException {
         String invalidJson = UUID.randomUUID().toString();
 
         Mockito.when(httpClient.execute(any(HttpUriRequest.class), any(ResponseHandler.class)))
-               .thenReturn(invalidJson);
```
Comment (Nurkambay) [inline src/test/java/rest/JsonRequestTest.java:54]: Please revert these changes.

File: src/test/java/rest/JsonRequestTest.java [hunk 5]
```diff
@@ -62,12 +61,37 @@ public void tstHttpGetReturnsJson() throws IOException {
         String json = "{\"data\":{}}";
 
         Mockito.when(httpClient.execute(any(HttpUriRequest.class), any(ResponseHandler.class)))
-               .thenReturn(json);
+            .thenReturn(json);
 
         JSONObject jsonObject = request.httpGet();
-        Mockito.verify(httpClient).execute(any(HttpGet.class), any(BasicResponseHandler.class));
```
Comment (Nurkambay) [inline src/test/java/rest/JsonRequestTest.java:68]: Please revert these changes.

File: src/test/java/rest/JsonRequestTest.java [hunk 6]
```diff
@@ -51,8 +53,7 @@ public void tstHttpGetReturnsNullForInvalidJson() throws IOException {
         String invalidJson = UUID.randomUUID().toString();
 
         Mockito.when(httpClient.execute(any(HttpUriRequest.class), any(ResponseHandler.class)))
-               .thenReturn(invalidJson);
-        Mockito.verify(httpClient, Mockito.times(0)).execute(any(HttpGet.class), any(BasicResponseHandler.class));
```
Comment (Nurkambay) [inline src/test/java/rest/JsonRequestTest.java:55]: This also was removed during resolving merge conflicts, please revert

File: src/test/java/rest/JsonRequestTest.java [hunk 7]
```diff
@@ -21,26 +20,29 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 
-@RunWith(MockitoJUnitRunner.class)
 public class JsonRequestTest {
 
     private HttpClient httpClient;
     private JsonRequest request;
 
-    @Before
```
Comment (Nurkambay) [inline src/test/java/rest/JsonRequestTest.java:30]: This also was removed during resolving merge conflicts, please revert
